scripts/r.unpack/r.unpack.py

```diff
@@ -73,10 +73,9 @@ def main():
     os.chdir(tmp_dir)
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
-        data_names = [
-            tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name
-        ]
-    except:
+        data_names = [tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name]
```

Comment:
Maybe a different black settings, but this line is too long and is thus not Black conformant. You can see it in the Black check. In any case, it is not really being modified by this PR, so even the white space should not change.